src/components/breadcrumb/__tests__/utils.test.js

```diff
@@ -0,0 +1,25 @@
+import { createUniqueCrumbs } from '../utils.js';
+
+describe('createUniqueCrumbs', () => {
+  test('unique titles', () => {
+    expect(
+      createUniqueCrumbs([
+        {
+          title: 'Maps service',
+          path: '/api/maps/'
+        },
+        {
+          title: 'Overview',
+          path: '/api/'
+        },
+        {
+          title: 'API',
+          path: '/api/'
+        }
+      ])
+    ).toEqual([
+      { path: '/api/maps/', title: 'Maps service' },
+      { path: '/api/', title: 'Overview' }
```

Comment:
If there are two titles with the same path, will this utility always return the first one? I wonder if we might want to throw instead to alert contributors that they have included two of the same path. Or is there another scenario you have in mind where we would expect two of the same path to make it here?

Reply:
If there are two titles with the same path, the utility should return the first one. It should be a rare case, but when I was making test cases it did show up. Ideally, the logic should be builtin to PageLayout so the contributor shouldn't have to worry about configuring this component at all. It may take some trial and error during the testing phase to identify any one offs and then create tests cases, and update the logic.

My only concern with throwing an alert is that unless it breaks the build, then it can be really easy for the contributor to miss the alert. It could be worth it to think this concept of alerts.